bubble_sort.rb

```diff
@@ -0,0 +1,49 @@
+def is_sorted(ar)
+  length_of_array = ar.length
+  i = 0
+  while i < length_of_array do
```

Comment:
You can omit the `do` here. (While is a keyword. Works the same was as an if statement)